conf.d/metrics.conf

```diff
@@ -1,7 +1,7 @@
 [program:collector]
 command=twistd --nodaemon
     --pidfile=%(program_name)s.pid vumi_worker
-    --worker-class vumi.blinkenlights.GraphiteMetricsCollector
+    --worker-class vumi.blinkenlights.UDPMetricsCollector
     --hostname %(ENV_AMQP_HOST)s
     --port %(ENV_AMQP_PORT)s
     --vhost %(ENV_AMQP_VHOST)s
```

Comment:
The `--set-option=metrics_host:localhost` and `--set-option=metrics_port:9109` args need to be in this block.

Reply:
okay, thanks